twisted/internet/test/test_inotify.py

```diff
@@ -128,8 +127,7 @@ def test_closeWrite(self):
         callback.
         """
         def operation(path):
-            fObj = path.open("w")
-            fObj.close()
+            path.open("w").close()
```

Comment:
@markrwilliams like the message for the commit which this change is part of says, "Make trivial open/close operations oneliners for easier greppability". So it's easier to grep for example `open` invocations without `def`, `with`, `close` etc when looking for files that may not be explicitly closed.